src/reduxTypes.js

```diff
@@ -482,6 +482,24 @@ export function assumeSecretlyGlobalState(state: PerAccountState): GlobalState {
   return state;
 }
 
+/**
+ * Use the given state object as a per-account state.
+ *
+ * TODO(#5006): We'll have to fix and eliminate each call to this.
+ */
+export function dubPerAccountState(state: GlobalState): PerAccountState {
```

Comment:
It basically is. I had a distinction in my mind when I picked a different name here... but I think at the end of this PR, where `GlobalState` is no longer a subtype of `PerAccountState` any more than vice versa, the distinction no longer means anything, so probably it wasn't helpful in the first place.

(I think the idea was that because at the time `GlobalState <: PerAccountState`, it wasn't "secret" but openly there in the types that a `state: GlobalState` was in fact a `PerAccountState`, and the function was just a way of making that conversion explicitly.)